### DISCUSSION TITLE
"lastmod" keeps getting updated with each new blog post?

### DISCUSSION THREAD

**Discussion by talonx** [facebook/docusaurus#10703] [Q&A]
I have a Docusaurus deployment in blog only mode. Each time I publish a new post the last mod of _all_ entries in the sitemap.xml get updated. Is there a way to control this?

My sitemap config looks like this:
```
        sitemap: {
          lastmod: 'date',
          changefreq: 'weekly',
          priority: 0.5,
          ignorePatterns: [
            '/tags/**',
            '/markdown-page',
            '/authors',
          ],
          filename: 'sitemap.xml',
          createSitemapItems: async (params) => {
            const { defaultCreateSitemapItems, ...rest } = params;
            const items = await defaultCreateSitemapItems(rest);
            return items.filter((item) => !item.url.includes('/page/'));
          },
        },

```

**Comment by slorber** [comment]
The system that performs the deployment should not use a shallow Git clone otherwise our `git log` commands will only see the last git commit date for all files.

Probably related to https://github.com/facebook/docusaurus/issues/10031
